go.mod

```diff
@@ -1,14 +1,8 @@
-module github.com/keithalucas/longhorn-spdk
+module github.com/longhorn/longhorn-spdk
 
 go 1.13
```

Comment:
Let's update the go version to v1.17 same as in the manager.
This also changes the way modules are handled a bit.

Reply:
Updated.